packages/core/content-releases/admin/src/components/CMReleasesContainer.tsx

```diff
@@ -250,15 +251,16 @@ export const CMReleasesContainer = () => {
   const { formatMessage } = useIntl();
   const {
     isCreatingEntry,
-    allLayoutData: { contentType },
+    hasDraftAndPublish,
+    initialData: { id: entryId },
```

Comment:
This is what I mean in the other PR @Marc-Roig, here I can get the id of the singleType (they have an unique id in the singleType database for each locale, and if you delete the entry and create a new one the id increments) then I can use this id for `findOne` on the backend..

With locale was impossible to me to get singleTypes from locales different than default one